### ISSUE TITLE
numpy package from this repo is broken

### ISSUE THREAD

**Issue by chumaroz** [apple/tensorflow_macos#207]
# Reproducing code example:
```python
import numpy as np
np.isnan(np.nan)
```

# Error message:
the above code results in False.
In other words, nan values are not recognized as nan.

# NumPy/Python version information:
1.18.5 3.8.8 (default, Feb 26 2021, 09:10:27)
[Clang 12.0.0 (clang-1200.0.32.29)]

**Comment by hannesa2** [comment]
It works for me

```
(tensorflow_venv) ➜  city-simulator git:(master) ✗ python
Python 3.8.8 (default, Feb 26 2021, 09:10:27) 
[Clang 12.0.0 (clang-1200.0.32.29)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import numpy as np
np.isnan(np.nan)
>>> np.isnan(np.nan)
False

```

**Comment by chumaroz** [comment]
> It works for me
> 
> ```
> (tensorflow_venv) ➜  city-simulator git:(master) ✗ python
> Python 3.8.8 (default, Feb 26 2021, 09:10:27) 
> [Clang 12.0.0 (clang-1200.0.32.29)] on darwin
> Type "help", "copyright", "credits" or "license" for more information.
> >>> import numpy as np
> np.isnan(np.nan)
> >>> np.isnan(np.nan)
> False
> ```

@hannesa2 
Yes that just proves the bug exists. And that behaviour can cause multiple issue in dependent modules.

**Comment by MikeTuomas** [comment]
Your example works for me with conda installation https://github.com/apple/tensorflow_macos/issues/153 .

**Comment by hannesa2** [comment]
I can confirm that conda is working 

<img width="588" alt="image" src="https://user-images.githubusercontent.com/3314607/112441141-64509b80-8d4b-11eb-949c-28c1f81ee3e4.png">

**Comment by StefanStsc** [comment]
I can confirm that numpy is broken for me as well

**Comment by jeffreyrdcs** [comment]
I can confirm that the numpy that comes with the script is broken.  I can reproduce the above error.
It also breaks the pandas.describe() and causes seg fault.
